pkg/components/components.go

```diff
@@ -179,105 +182,10 @@ func GetDeploymentSpecOperator(params *DeploymentOperatorParams) appsv1.Deployme
 						Name:            hcoName,
 						Image:           params.Image,
 						ImagePullPolicy: corev1.PullPolicy(params.ImagePullPolicy),
-						// command being name is artifact of operator-sdk usage
-						Command:        stringListToSlice(hcoName),
-						ReadinessProbe: getReadinessProbe(),
-						LivenessProbe:  getLivenessProbe(),
-						Env: append([]corev1.EnvVar{
-							{
-								// deprecated: left here for CI test.
-								Name:  util.OperatorWebhookModeEnv,
-								Value: "false",
-							},
-							{
-								Name:  util.ContainerAppName,
-								Value: util.ContainerOperatorApp,
-							},
-							{
-								Name:  "KVM_EMULATION",
-								Value: "",
-							},
-							{
-								Name:  "OPERATOR_IMAGE",
-								Value: params.Image,
-							},
-							{
-								Name:  "OPERATOR_NAME",
-								Value: hcoName,
-							},
-							{
-								Name:  "OPERATOR_NAMESPACE",
-								Value: params.Namespace,
-							},
-							{
-								Name: "POD_NAME",
-								ValueFrom: &corev1.EnvVarSource{
-									FieldRef: &corev1.ObjectFieldSelector{
-										FieldPath: "metadata.name",
-									},
-								},
-							},
-							{
-								Name:  "VIRTIOWIN_CONTAINER",
-								Value: params.VirtIOWinContainer,
-							},
-							{
-								Name:  "SMBIOS",
-								Value: params.Smbios,
-							},
-							{
-								Name:  "MACHINETYPE",
-								Value: params.Machinetype,
-							},
-							{
-								Name:  "AMD64_MACHINETYPE",
-								Value: params.Amd64MachineType,
-							},
-							{
-								Name:  "ARM64_MACHINETYPE",
-								Value: params.Arm64MachineType,
-							},
-							{
-								Name:  util.HcoKvIoVersionName,
-								Value: params.HcoKvIoVersion,
-							},
-							{
-								Name:  util.KubevirtVersionEnvV,
-								Value: params.KubevirtVersion,
-							},
-							{
-								Name:  util.CdiVersionEnvV,
-								Value: params.CdiVersion,
-							},
-							{
-								Name:  util.CnaoVersionEnvV,
-								Value: params.CnaoVersion,
-							},
-							{
-								Name:  util.SspVersionEnvV,
-								Value: params.SspVersion,
-							},
-							{
-								Name:  util.HppoVersionEnvV,
-								Value: params.HppoVersion,
-							},
-							{
-								Name:  util.AaqVersionEnvV,
-								Value: params.AaqVersion,
-							},
-							{
-								Name:  util.KVUIPluginImageEnvV,
-								Value: params.KVUIPluginImage,
-							},
-							{
-								Name:  util.KVUIProxyImageEnvV,
-								Value: params.KVUIProxyImage,
-							},
-							{
-								Name:  util.PrimaryUDNImageEnvV,
-								Value: params.PrimaryUDNImage,
-							},
-						}, params.Env...),
```

Comment:
Please preserve this logic in the new function: adding the `Env` param values to the end of the slice.